### DISCUSSION TITLE
SynEdit Update calls.

### DISCUSSION THREAD

**Discussion by MShark67** [TurboPack/SynEdit#176] [General]
When I converted the Scrollbar code, I kept the existing Update calls as I didn't want to change any behavior (perhaps ones I didn't understand.)  Does anyone know why these are there?  They cause issues if you need to load a number of SynEdits in a loop (basically each one will force paint itself, causing slowdowns and flashing.)  Removing the calls fixes that issue and I haven't noticed any problems with removing them, but they must be there for a reason I don't know about.

**Comment by pyscripter** [comment]
I did try removing it a long time ago, but I then decided to put it back.   I really cannot remember why.   If you do not updated at that point,  the invalidated area will be painted at the next paint message, so removing it might be OK and it would be good,  but It needs more testing.

A solution to your issue might be to send a WM_SETREDRAW  message to the parent window before and after loading your SynEdits.

**Comment by pyscripter** [comment]
I did some limited testing and could not find anything wrong with removing it.   I have created an issue about this (#178 ).  Please get back to me if when you are convinced that removing it does not break anything.
